integration-tests/fromMnemonic-send-tez.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { OperationContentsAndResultTransaction } from "@taquito/rpc";
+import { InMemorySigner } from "@taquito/signer";
+import { TezosToolkit } from "@taquito/taquito";
+import { CONFIGS } from "./config";
+import * as Bip39 from 'bip39'
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  let Funder: TezosToolkit;
+   describe(`Test fromMnemonic instantiation with rpc: ${rpc}`, () => {
+    let mnemonic: string;
+
+    beforeEach(async (done) => {
+      mnemonic = Bip39.generateMnemonic();
```

Comment:
do we need to generate mnemonic for each `it` block? If not, we could instead use `beforeAll`